openslides/core/models.py

```diff
@@ -243,3 +273,61 @@ class Meta:
     def get_session_store_class(cls):
         from .session_backend import SessionStore
         return SessionStore
+
+
+class Message(RESTModelMixin, models.Model):
```

Comment:
But with the motions, you can remove them with purge_elements. This is not possible for Messages and Countdowns, and with saving the uuid there is a direct access to these elements without searching for them.